src/gtfs/write.rs

```diff
@@ -278,6 +283,108 @@ pub fn write_stop_extensions(
     Ok(())
 }
 
+#[derive(Debug)]
+struct PhysicalModeWithOrder<'a> {
+    inner: &'a objects::PhysicalMode,
+    is_lowest: bool,
+}
+
+fn get_line_physical_modes<'a>(
+    idx: Idx<objects::Line>,
+    collection: &'a CollectionWithId<objects::PhysicalMode>,
+    model: &Model,
+) -> Vec<PhysicalModeWithOrder<'a>>
+where
+    IdxSet<objects::Line>: GetCorresponding<objects::PhysicalMode>,
+{
+    let mut pms: Vec<&objects::PhysicalMode> = model
+        .get_corresponding_from_idx(idx)
+        .into_iter()
+        .map(move |idx| &collection[idx])
+        .collect();
+    pms.sort_unstable_by_key(|pm| get_physical_mode_order(pm));
+
+    pms.iter()
+        .enumerate()
+        .map(|(i, pm)| PhysicalModeWithOrder {
+            inner: pm,
+            is_lowest: i == 0,
+        }).collect()
+}
+
+impl<'a> From<&'a objects::PhysicalMode> for RouteType {
+    fn from(obj: &objects::PhysicalMode) -> RouteType {
+        match obj.id.as_str() {
```

Comment:
@prhod questions are linked to what we decided here : https://jira.kisio.org/browse/NAVP-1007
1. Looks like nothing maps to RouteType::CableCar (and I'm curious of the opposite also : cable car should be mapped to funicular?).
2. Same question for Gondola_SuspendedCableCar (and we mentionned creation of SuspendedCableCar in NTFS for that)

Also, some types are missing (Taxi and Air if I'm correct).

It would be nice to have the reverse mapping done also here (even if not used) to compare.

Reply:
After discussion with @prhod on the 2 questions I had:
1. GTFS type 5 (CableCar) is to be mapped to Funicular in NTFS (so nothing to do here, maybe some doc)
2. Gondola_SuspendedCableCar will be mapped to SuspendedCableCar in NTFS, and it will have to be managed here, but not urgent.

Everything is tracked in Nav2's JIRA ticket https://jira.kisio.org/browse/NAVP-1007